google/cloud/internal/credentials.cc

```diff
@@ -36,6 +36,30 @@ std::shared_ptr<Credentials> MakeAccessTokenCredentials(
   return std::make_shared<AccessTokenConfig>(access_token, expiration);
 }
 
+std::shared_ptr<ImpersonateServiceAccountConfig>
+MakeImpersonateServiceAccountCredentials(
+    std::shared_ptr<Credentials> base_credentials,
+    std::string target_service_account, Options opts) {
+  opts = MergeOptions(
+      std::move(opts),
+      Options{}
+          .set<LifetimeOption>(std::chrono::seconds(std::chrono::hours(1)))
+          .set<ScopesOption>({"https://www.googleapis.com/auth/cloud-platform"})
+          .set<DelegatesOption>({}));
```

Comment:
I think there should be no need to set an empty option like this, unless you're relying on `.has<T>()` always returning true. IIRC, `.get<T>()` and `.lookup<T>()` should do the Right Thing, even if this option didn't previously exist.

Reply:
Fixed.